ComSemApp/models.py

```diff
@@ -110,6 +110,12 @@ def get_visible_worksheets(self):
         self.worksheets : QuerySet[Worksheet]
         return self.worksheets.exclude(status=teacher_constants.WORKSHEET_STATUS_PENDING)
 
+    def get_speaking_practice_review_requests(self):
+        requests = SpeakingPracticeAttemptReviewRequest.objects.filter(
+            attempt__expression__worksheet__course=self
+        )
+        return requests
```

Comment:
Could just return the right side of the assignment statement rather than assigning and then returning